client/my-sites/earn/home.tsx

```diff
@@ -300,7 +306,7 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 			image: {
 				path: earnSectionImage,
 			},
-			body: translate( 'There is a range of ways to earn money through your WordPress site.' ),
+			body: translate( 'Turn your website into a reliable source of income.' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Turn your website into a source of income.' )` *ES Score: 7*